README.md

```diff
@@ -15,6 +15,9 @@ You can import the billing object anywhere in your application:
 import billing as billing;
 ~~~
 
+### facebook web Setup
+
+For facebook iap to integrate with your game make sure the `product id` is same in both app dashboard and game.
```

Comment:
Please add steps on how to configure IAP in the game

Reply:
We don't have to @rampr as there is no change for facebook or device. Only product id needs to be same.